openlxp_authentication/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.db.models.base import Model
+from django.conf import settings
+from social_core.backends.saml import SAMLAuth, SAMLIdentityProvider
+from social_django.strategy import DjangoStrategy
+
+
+class SAMLConf(Model):
+    name = models.CharField(unique=True, max_length=50)
```

Comment:
We should really have a doc string describing the model

Reply:
Added